SPECS/dracut/90overlayfs/azl-configure-selinux.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+type getarg > /dev/null 2>&1 || . /lib/dracut-lib.sh
+
+# If SELinux is disabled exit now
+getarg "selinux=0" > /dev/null && return 0
+
+SELINUX="enforcing"
+# shellcheck disable=SC1090
+[ -e "$NEWROOT/etc/selinux/config" ] && . "$NEWROOT/etc/selinux/config"
+[ "$SELINUX" == "disabled" ] && return 0
+
+getargbool 0 rd.live.overlay.overlayfs && overlayfs="yes"
+
+if [ -n "$overlayfs" ]; then
+    [ -e /sysroot ] && chcon -t root_t /sysroot
+    [ -e /run/overlayfs ] && chcon -t root_t /run/overlayfs
+    [ -e /run/ovlwork ] && chcon -t root_t /run/ovlwork
```

Comment:
Hard coding `root_t` is undesirable if we can avoid it... that being said, it's incredibly unlikely it would change.

You could try to get the context from `matchpathcon -f $NEWROOT/etc/selinux/targeted/contexts/files/file_contexts -m dir /`, but I don't know if that command would be available.

Reply:
we can use matchpathcon from the rootfs. I've updated the code to dynamically determine the label and use it.